changes/ee/fix-16105.en.md

```diff
@@ -0,0 +1 @@
+Improve the performance of the connection process of the durable sessions.
```

Comment:
maybe make it more clear from end-user pov. what difference is it to be expected, would be even better how much different etc.